### PR TITLE
LDR overview and monitoring pages

### PR DESCRIPTION
**Pull request by kathancox**
This PR adds the Monitoring page and General Overview page for the LDR section.

## Preview

[Overview page](https://deploy-preview-19085--cockroachdb-docs.netlify.app/docs/v24.3/logical-data-replication-overview.html)
[Monitoring page](https://deploy-preview-19085--cockroachdb-docs.netlify.app/docs/v24.3/logical-data-replication-monitoring.html)

This PR is 2 of 4(?) for LDR docs. It contains:

Overview: General description of LDR and its features/use cases
Monitoring LDR: This is a page that describes what is available for monitoring and links out to the relevant docs material

Reviewers: You'll see a lot of comments for follow-up PRs to connect pages with links and such.

**Content pages to come:**
CREATE/SHOW sql ref
Technical Overview
LDR Metrics Dashboard docs

PR 1 #19043

### CODE REVIEW

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:11]: how does this work: 
Both source and destination cluster can receive application reads and writes, with LDR enabling bidirectional replication for eventual consistency in the replicating tables. 

This enables an active-active setup which can provide...

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:14]: Should we say "If you are looking cross-cluster replication that is transactionally consistent..."

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:19]: Unfinished sentence?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:23]: nit: particularly during region or data-center failures?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:21]: Instead of calling out the way that LDR is run, should we just have the title be use cases? "Achieve high availability and single-region write latency with 2DC deployments" 

and then, "Achieve workload isolation between clusters"?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:38]: "LDR offers different modes which apply data differently during replication, allowing the user to make tradeoffs between optimizing for throughput vs. constraints during replication" thoughts on this?

File: src/current/_includes/v24.3/ldr/multiple-tables.md
Comment (alicia-l2) [inline src/current/_includes/v24.3/ldr/multiple-tables.md:1]: I feel like this note combines operational consideration with observability. Should we keep it scoped to observability considerations given that this appears on the observability page only?

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-monitoring.md:25]: nit: When you start an LDR stream?

File: src/current/_includes/v24.3/ldr/show-logical-replication-responses.md
Comment (alicia-l2) [inline src/current/_includes/v24.3/ldr/show-logical-replication-responses.md:1]: Is this missing the "description" field in LDR responses?

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-monitoring.md:64]: "This is tracked from when a row is committed on the source cluster, to when it is "committed" on the target cluster. A commit on the target cluster is when".... 

could we add this sentence in after "The commit-to-commit replication latency"

**Review by alicia-l2 — APPROVED** [review]
Left some comments, LGTM

File: src/current/v24.3/set-up-logical-data-replication.md
Comment (alicia-l2) [inline src/current/v24.3/set-up-logical-data-replication.md:60]: SQL constraints --> it should be "you cannot replicate a table with foreign key constraints". Other constraints are fine.

File: src/current/_includes/v24.3/ldr/multiple-tables.md
Comment (kathancox) [reply src/current/_includes/v24.3/ldr/multiple-tables.md:1]: So, I have now added this "include" file to the tutorial page as well. It just ensures that if we change the advice in any way, I can update this one file and it will update anywhere. I felt like it might be ok on the monitoring page, as it is also talking about job operation (which we kind of touch on in Monitoring LDR).

File: src/current/_includes/v24.3/ldr/show-logical-replication-responses.md
Comment (kathancox) [reply src/current/_includes/v24.3/ldr/show-logical-replication-responses.md:1]: yes it is, I am adding. Thanks for catching.

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:11]: > The active-active setup can provide protection against cluster,

The active-active setup between clusters?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:14]: continually replicates data for transactional consistency from a?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:26]: "Maintain high availability and be resilient to region failures with..."?

Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:26]: > Both clusters can receive application reads and writes with low, single-region write latency. 

can we start this with "If you set up two single-region clusters, in LDR, both clusters can receive..."

>  In a datacenter or cluster outage, you can redirect application traffic to the surviving cluster with 

"Then, in a datacenter, region, or cluster outage"?

>  In the following diagram, the clusters are deployed in US East and West to provide low latency for that region. 

the two single-region clusters?

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-overview.md:28]: small nit -- on top of the arrows can we write LDR stream #1 and LDR stream #2

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (alicia-l2) [inline src/current/v24.3/logical-data-replication-monitoring.md:135]: is this WITH option included in the setup page too?

**Review by alicia-l2 — APPROVED** [review]
LGTM added some small comments. thank you!

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Done

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Done

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Done

Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Done

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-monitoring.md:135]: I need to add this in a few places

File: src/current/v24.3/logical-data-replication-overview.md
Comment (msbutler) [inline src/current/v24.3/logical-data-replication-overview.md:32]: again, i could see bidi deployments for a workload isolation use case. (hot cold)

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (msbutler) [inline src/current/v24.3/logical-data-replication-monitoring.md:60]: ugh, i may have to backport some changes to this table. this is fine to merge as is.

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (msbutler) [inline src/current/v24.3/logical-data-replication-monitoring.md:68]: replication lag is not provided by the metric `logical_replication.replicated_time_seconds`. Rather, the user has to compute  time.now()-replicated_time_seconds to get the replication lag.

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (msbutler) [inline src/current/v24.3/logical-data-replication-monitoring.md:72]: i think we should only document `logical_replication.events_dlqed`, which you do in the line above. i think these metrics below are probably only for crdb internal use for now. I would remove them from the docs for now.

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (msbutler) [inline src/current/v24.3/logical-data-replication-monitoring.md:122]: while we're here, you could add the .scanning_ranges and .lagging_ranges metrics. added on friday https://github.com/cockroachdb/cockroach/pull/134674/commits/9eb6c8bd73b4d532ca0ed3170a601086e14850c9

**Review by msbutler — COMMENTED** [review]
looks great! left mostly nits

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [reply src/current/v24.3/logical-data-replication-overview.md:26]: We can link to blog posts describing this for the first use case.
For second use case, @kathancox maybe we can try to get more specific about hardware/cluster specific isolation? 

maybe above both use cases we should add a note saying that we consider this a tool and is an alternative deployment option to our native Raft/MRarchitecture

File: src/current/v24.3/logical-data-replication-overview.md
Comment (alicia-l2) [reply src/current/v24.3/logical-data-replication-overview.md:32]: True, maybe we dont need to call out that its unidirectional explicitly

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md:26]: Should we just link to the Data Resilience page? I think that covers some of this, particularly when it comes to MR feature comparison. I'm adding a note linking to that now.

Anything further in terms of execution locality + cdc comps, we should probably create another docs issue for that.

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Altered this a bit so it's clear "source" ---> "destination". Otherwise, I think the vocabulary itself explains this.

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-monitoring.md:118]: Thanks, I have added these to this label list

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-monitoring.md:68]: I have added this in

File: src/current/v24.3/logical-data-replication-overview.md
Comment (msbutler) [reply src/current/v24.3/logical-data-replication-overview.md:30]: yeah, i think we should revisit how we publicly document the pros and cons of LDR + PCR + CRDB, but i don't think we need to block this PR.

**Review by msbutler — APPROVED** [review]

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (rmloveland) [inline src/current/v24.3/logical-data-replication-monitoring.md:64]: I don't like the scare quotes around "committed" and i don't think overloading the term commit is a good idea

COMMIT is already very well defined semantically in SQL world and in our docs

i'd actually reverse this and say "to when it is applied on the destination cluster" where "applied" means either:
- COMMIT to target table, OR
- inserted to DLQ (which is also a COMMIT)

pls ignore for now if there isn't time for this but i think it's a serious problem if we are sprinkling "committed" to mean "LDR's special notion of committed" around LDR docs when we really mean "applied" per the definition i'm using above which doesn't cause the same confusion IMO

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (rmloveland) [inline src/current/v24.3/logical-data-replication-monitoring.md:80]: the "either" is dangling, needs a second case

or did you mean "will show the avg metrics across"

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (rmloveland) [inline src/current/v24.3/logical-data-replication-monitoring.md:84]: suggest linking to metrics dashboard again for people like me with no short term memory

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (rmloveland) [inline src/current/v24.3/logical-data-replication-monitoring.md:101]: same as above re: recommend linking to jobs page docs for we who live the memento lifestyle

**Review by rmloveland — APPROVED** [review]
LGTM modulo that i strongly suggest if at all possible switching from scare quote "committed" to "applied" and then defining what "applied" means (see my comment) since COMMIT is already super duper in use and we should not be redefining it, it will only lead to tears IMO

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md:30]: Done!

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Done!

File: src/current/v24.3/logical-data-replication-overview.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-overview.md]: Changed to view

File: src/current/_includes/v24.3/ldr/show-logical-replication-responses.md
Comment (kathancox) [reply src/current/_includes/v24.3/ldr/show-logical-replication-responses.md]: done

File: src/current/v24.3/logical-data-replication-monitoring.md
Comment (kathancox) [reply src/current/v24.3/logical-data-replication-monitoring.md]: done!

**Comment by kathancox** [comment]
TFTRs!

**Pull request merged**
